docs/partials/document-domain-workaround.md

```diff
@@ -0,0 +1,13 @@
+<Alert type="warning">
```

Comment:
these old alerts won't work. Replace this with an :::caution admonition

Reply:
I think I have this fixed in 74416b67c. Does everything look right on your end?